chainer/training/extensions/log_report.py

```diff
@@ -91,7 +91,11 @@ def __call__(self, trainer):
                 fd, path = tempfile.mkstemp(prefix=log_name, dir=trainer.out)
                 with os.fdopen(fd, 'w') as f:
                     json.dump(self._log, f, indent=4)
-                os.rename(path, os.path.join(trainer.out, log_name))
+
+                log_path = os.path.join(trainer.out, log_name)
+                if os.path.exists(log_path):
+                    os.remove(log_path)
+                os.rename(path, log_path)
```

Comment:
Thank you for the suggestion. I guess `shutil.move` cannot solve the problem occurred in Windows. According to the document, if the destination exists and is not a directory, `os.move` is used to overwrite the destination. But on Windows, `os.move` raises `OSError` when the destination exists (regardless of the type of the destination).